### ISSUE TITLE
classList not working when specified before class

### ISSUE THREAD

**Issue by sickill** [solidjs/solid#649]
**Describe the bug**

The order of `class` and `classList` props matters.

**To Reproduce**

Create a component with both `class` and `classList`, with `classList` specified first:
```
<div classList={...} class="...">...</div>
```

Observe that classes from `classList` are ignored (or probably overwritten by `class`).

**Expected behavior**

The order of `class` and `classList` shouldn't matter, or it should be noted in the docs (e.g. here https://www.solidjs.com/docs/latest/api#classlist) that `classList` must go _after_ `class`.

When it's reversed all works fine, both classes from "static" `class` and "dynamic" `classList` are used.

**Additional context**

solid-js 1.1.2

**Comment by ryansolid** [comment]
Yeah that's true. I wonder if it is really best to automatically change the order. These do need to run in sequence though. It's much simpler for me to document this for now and consider the implications of re-ordering things later.

**Comment by ryansolid** [comment]
Oops closed the wrong issue. In any case, I'm going to send this over to docs.
